public/script.js

```diff
@@ -203,6 +203,7 @@ let dialogueResolve = null;
 let chat_metadata = {};
 let streamingProcessor = null;
 
+window.filterByFav = false;
```

Comment:
Hey Cohee. I was keeping it on window because I need to update the variable from the group-chats.js on line 846. 

When the user selects/enters the group chat for the first time, it would reset the character listing leaving the fav star filter icon lit with the filterByFav on true.